Control coverage/Feature/ApplicationGateway.md

```diff
@@ -0,0 +1,101 @@
+﻿# ApplicationGateway
+
+**Resource Type:** Microsoft.Network/applicationGateways
+
+<!-- TOC -->
+
+- [Azure_ApplicationGateway_NetSec_Enable_WAF_Configuration_Trial](#azure_applicationgateway_netsec_enable_waf_configuration_trial)
```

Comment:
Shall we remove trial now, as this control is going to be merged now?

Reply:
this is done.